packages/prosess-uttak/src/util/uttaksperioder.ts

```diff
@@ -14,7 +14,23 @@ const sjekkOmPerioderErKantIKant = (periode: Period, nestePeriode: Period) => {
   );
 };
 
-const lagUttaksperiodeliste = (uttaksperioder: Uttaksperioder): Uttaksperiode[] => {
+const injectInntektsgradering = (uttaksperioder, inntektsgraderinger) => {
+  uttaksperioder.forEach(uttaksperiode => {
+    uttaksperiode.inntektsgradering =
```

Comment:
Burde kanskje unngå å mutere denne, for den kommer vel fra et stykke opp i treet?

Reply:
Fra BehandlingPleiepengerIndex tror jeg

Reply:
Ingen verdier blir endret, kun en lagt til. Men godt poeng, endret